src/Atmos/Model/bc_energy.jl

```diff
@@ -1,10 +1,11 @@
 abstract type EnergyBC end
 
 using ..TurbulenceClosures
-
+using ClimateMachine.SurfaceFluxes:
+    get_energy_flux, surface_conditions, DGScheme
+using ClimateMachine.Atmos: altitude
```

Comment:
```suggestion
using ..SurfaceFluxes:
    get_energy_flux, surface_conditions, DGScheme
```
This file is included in AtmosModel.jl, so we don't need `using ClimateMachine.Atmos: altitude`